src-json/meta.json

```diff
@@ -1130,5 +1130,26 @@
 		"metadata": ":nativeArrayAccess",
 		"doc": "When used on an extern class which implements haxe.ArrayAccess native array access syntax will be generated",
 		"platforms": ["cpp"]
+	},
+	{
+		"name": "CppValueType",
+		"metadata": ":cpp.ValueType",
+		"doc": "",
+		"platforms": ["cpp"],
+		"targets": [ "TClass", "TAbstract" ]
+	},
+	{
+		"name": "CppPointerType",
+		"metadata": ":cpp.PointerType",
+		"doc": "",
+		"platforms": ["cpp"],
+		"targets": [ "TClass" ]
+	},
+	{
+		"name": "CppManagedType",
+		"metadata": ":cpp.ManagedType",
+		"doc": "",
+		"platforms": ["cpp"],
+		"targets": ["TClass"]
```

Comment:
Would be better to move those up with other `Cpp*` metadata, and add `doc` :)

Reply:
The other cpp meta seems to be a bit all over the place, but I've moved the new ones up to be next to a few others and filled in the doc comment.